ia-guide/ia-cyoa.tw

```diff
@@ -215,44 +264,280 @@ And now that they've managed to complete {reader3action2} what is the next thing
 
 {text input for: reader3action3}
 
-Great! Now you have the three main things that a reader like {readerName3} might want to do, it's time to put all these together.
+Great! Now you have the three main things that a reader like {readerName3} might want to do, [[go back->Why do they read the docs?]] and do the same for {readerName2} and {readerName1}.
+Or, if you're done, move on to [[Writing for your readers->Writing for your readers]]
+
+
+:: Writing for your readers
+critpath1:''
+critpath2:''
+critpath3:''
+critpath4:''
+critpath5:''
+critpath6:''
+critpath7:''
+critpath8:''
+critpath9:''
+critpath10:''
+critpath11:''
+critpath12:''
+critpath13:''
+critpath14:''
+critpath15:''
+critpath16:''
+critpath17:''
+critpath18:''
+critpath19:''
+critpath20:''
+critpath21:''
+critpath22:''
+critpath23:''
+critpath24:''
+critpath25:''
+critpath26:''
+critpath27:''
+reader1action1critscore: critpath1 + critpath2 + critpath3
+reader1action2critscore: critpath4 + critpath5 + critpath6
+reader1action3critscore: critpath7 + critpath8 + critpath9
+reader2action1critscore: critpath10 + critpath11 + critpath12
+reader2action2critscore: critpath13 + critpath14 + critpath15
+reader2action3critscore: critpath16 + critpath17 + critpath18
+reader3action1critscore: critpath19 + critpath20 + critpath21
+reader3action2critscore: critpath22 + critpath23 + critpath24
+reader3action3critscore: critpath25 + critpath26 + critpath27
+--
 
-[[Why do readers read docs?->why]]
+## Writing for your readers
+Now that you've had a think about who your readers are, and the kinds of things they might be using your documentation for, you can display them in a table like this:
+
+<table>
```

Comment:
Do you reckon this would translate well to Markdown or non-interactive mode?

Reply:
The way I see it, those fields just need to be blank for non-interactive mode. My thinking has been that we need to be able to just remove the interactive elements, and encourage use of a pen and paper.